src/utils/numToPrice.ts

```diff
@@ -5,10 +5,14 @@
   console.log(price); //4.80
 */
 export default function numToPrice(num: number): string {
+  num = Math.round(num * 1e2) / 1e2;
+
+  //num is integer //5 -> 5.00
+  if (num % 1 === 0) return num.toString().concat(".00");
+
   //num has 1 dec place //5.9 -> 5.90
   if ((num * 10) % 1 === 0) return num.toString().concat("0");
 
-  //num is whole number //5 -> 5
   //num has 2 dec place //5.99 -> 5.99
   return num.toString();
```

Comment:
What if it has more than 2dp(maybe there are percentage discounts or whatever)? Need to round off to 2d.p first